app/wama.web/AppString.resx

```diff
@@ -151,4 +151,7 @@
   <data name="PatronLabel" xml:space="preserve">
     <value>Patron</value>
   </data>
+  <data name="SignatureMismatch" xml:space="preserve">
+    <value>Signature does not match with the name of the Patron.</value>
```

Comment:
Actually, on a second thought, I think this might be much clear

Signature does not match expected format "Firstname Lastname"